common/entity/Person.ts

```diff
@@ -66,6 +66,16 @@ export abstract class Person {
         default: false,
     })
     isRedacted: boolean;
+
+    @Column({
+        type: 'timestamp',
+        nullable: true,
+        default: new Date(),
+    })
+    lastTimeCheckedNotifications: Date;
+
+    @Column({ nullable: true, type: 'json' })
+    notificationPreferences: any;
```

Comment:
why not string?

Reply:
Idk, I think this code might be from @Jonasdoubleyou. Maybe the type `NotificationPrefereces | null` could be used here, but I'm not sure.

Reply:
Ah yeah, a proper type could also be used here, was too lazy 😅